### ISSUE TITLE
Automatic calibration

### ISSUE THREAD

**Issue by osnr** [janpaul123/paperprograms#29]
Now that threshold and especially sigma might come in, it's becoming annoying.

Something where on boot, it tells you to put a piece of paper down, and it searches for parameters such that it can recognize that piece of paper?

**Comment by janpaul123** [comment]
Yeah, or a button to do that (ideally with any programs that are just scattered around)? Since it will remember settings it might not be always necessary on boot.
